pages/common/supervisorctl.md

```diff
@@ -0,0 +1,24 @@
+# supervisorctl
+
+> Supervisor is a client/server system that allows its users to control a number of processes on UNIX-like operating systems.
+> Supervisorctl is the command-line client piece of the supervisor which provides a shell-like interface.
+
+- Start/stop/restart a process:
+
+`supervisorctl {{start|stop|restart}} {{process name}}`
+
+- Start/stop/restart all processes in a group:
+
+`supervisorctl {{start|stop|restart}} {{group name}}:*`
+
+- Show last N **bytes** of process stderr:
```

Comment:
Normally tools like tail shows last N lines by default, supervisor shows last N bytes, it's a common mistake people will take when they use it for first time, so I make it bold.

Change N to concrete number is a good idea, I will update this.